eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -16,7 +16,15 @@ steps:
     retryCountOnTaskFailure: 3
     inputs:
       # We must install sdk, not just runtime, as it is required by some of our tools, like test-proxy.
-      # For additional context, see PR: https://github.com/Azure/azure-sdk-tools/pull/5405
+      # Specifically, test-proxy requires asp.net core runtime, which is installed only when sdk option
+      # is selected, per: https://github.com/microsoft/azure-pipelines-tasks/issues/14405
+      #
+      # Build showing the lack of asp.net core runtime when 'runtime' is used instead of 'sdk':
```

Comment:
This link is probably not helpful as the build will go ahead after some time. I think the note above is more important.

Reply:
> as the build will go ahead

You mean `expire` here right? Agreed.

Reply:
yes the build will expire and the link will be broken.

Reply:
Replied [here](https://github.com/Azure/azure-sdk-tools/pull/5428#issuecomment-1428808128)